780 content types field complete requirements

nick-deer opened the pull request:
Addresses https://github.com/MukurtuCMS/Mukurtu-CMS/issues/780

nick-deer committed:
Limit to 1 media item on word lists

nick-deer committed:
Added location taxonomy field to person record form

nick-deer committed:
Add local contexts fields to person records

nick-deer committed:
Add local contexts fields to word lists

nick-deer committed:
Add local contexts fields to collections

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr785-xwdkoxhlb3rpqldrj0brejfyweakrsml.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67ec6e6f908d23476b793b4a

nick-deer commented:
@michael-wynne-wsu this is almost ready to go except for the location field renaming issue--could I get your input on which field to rename and what to rename it to?

michael-wynne-wsu commented:
@nick-deer I'd like to keep the taxonomy field as "location". 

For the mapping field, how about "map points" or "location data" - any reason for/against either of those?

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr785-xwdkoxhlb3rpqldrj0brejfyweakrsml.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67ec6e6f908d23476b793b4a

nick-deer commented:
> @nick-deer I'd like to keep the taxonomy field as "location".
> 
> For the mapping field, how about "map points" or "location data" - any reason for/against either of those?


I like both, but I think I'm more partial to "map points" since it's more distinguished from "location." I can go right ahead with that if you're okay with it!

nick-deer committed:
Add local contexts as dependency to pass tests

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr785-xwdkoxhlb3rpqldrj0brejfyweakrsml.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67ec6e6f908d23476b793b4a

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr785-xwdkoxhlb3rpqldrj0brejfyweakrsml.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67ec6e6f908d23476b793b4a

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr785-xwdkoxhlb3rpqldrj0brejfyweakrsml.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67ec6e6f908d23476b793b4a

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr785-xwdkoxhlb3rpqldrj0brejfyweakrsml.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67ec6e6f908d23476b793b4a

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr785-xwdkoxhlb3rpqldrj0brejfyweakrsml.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67ec6e6f908d23476b793b4a

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr785-xwdkoxhlb3rpqldrj0brejfyweakrsml.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67ec6e6f908d23476b793b4a

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr785-xwdkoxhlb3rpqldrj0brejfyweakrsml.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67ec6e6f908d23476b793b4a

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr785-xwdkoxhlb3rpqldrj0brejfyweakrsml.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67ec6e6f908d23476b793b4a

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr785-xwdkoxhlb3rpqldrj0brejfyweakrsml.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67ec6e6f908d23476b793b4a

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr785-xwdkoxhlb3rpqldrj0brejfyweakrsml.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67ec6e6f908d23476b793b4a

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr785-xwdkoxhlb3rpqldrj0brejfyweakrsml.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67ec6e6f908d23476b793b4a

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr785-xwdkoxhlb3rpqldrj0brejfyweakrsml.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67ec6e6f908d23476b793b4a

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr785-xwdkoxhlb3rpqldrj0brejfyweakrsml.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67ec6e6f908d23476b793b4a

michael-wynne-wsu commented:
Yes, map points is good!

nick-deer committed:
Merge branch 'main' into 780-content-types-field-complete-requirements

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr785-xwdkoxhlb3rpqldrj0brejfyweakrsml.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67ec6e6f908d23476b793b4a

nick-deer committed:
Merge branch 'main' into 780-content-types-field-complete-requirements

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr785-xwdkoxhlb3rpqldrj0brejfyweakrsml.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67ec6e6f908d23476b793b4a

nick-deer committed:
Rename geofield location field to 'Map Points'

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr785-xwdkoxhlb3rpqldrj0brejfyweakrsml.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67ec6e6f908d23476b793b4a

nick-deer committed:
Rename geofield location field to 'Map Points'

nick-deer committed:
Rename geofield location field to 'Map Points'

nick-deer committed:
Rename geofield location field to 'Map Points'

nick-deer committed:
Rename geofield location field to 'Map Points'

nick-deer committed:
Rename geofield location field to 'Map Points'

nick-deer committed:
Rename geofield location field to 'Map Points'

nick-deer committed:
Rename geofield location field to 'Map Points'

I know we have tabled work on Solr for now, but I wanted to be consistent with this renaming.

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr785-xwdkoxhlb3rpqldrj0brejfyweakrsml.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67ec6e6f908d23476b793b4a
